doc/man3/BIO_s_dgram_pair.pod

```diff
@@ -0,0 +1,220 @@
+=pod
+
+=head1 NAME
+
+BIO_s_dgram_pair, BIO_new_bio_dgram_pair, BIO_dgram_set_no_trunc,
+BIO_dgram_get_no_trunc, BIO_dgram_get_effective_caps, BIO_dgram_get_caps,
+BIO_dgram_set_caps, BIO_dgram_set_mtu, BIO_dgram_get_mtu - datagram pair BIO
+
+=head1 SYNOPSIS
+
+ #include <openssl/bio.h>
+
+ const BIO_METHOD *BIO_s_dgram_pair(void);
+
+ int BIO_new_bio_dgram_pair(BIO **bio1, size_t writebuf1,
+                            BIO **bio2, size_t writebuf2);
+ int BIO_dgram_set_no_trunc(BIO *bio, int enable);
+ int BIO_dgram_get_no_trunc(BIO *bio);
+ uint32_t BIO_dgram_get_effective_caps(BIO *bio);
+ uint32_t BIO_dgram_get_caps(BIO *bio);
+ int BIO_dgram_set_caps(BIO *bio, uint32_t caps);
+ int BIO_dgram_set_mtu(BIO *bio, unsigned int mtu);
+ unsigned int BIO_dgram_get_mtu(BIO *bio);
+
+=head1 DESCRIPTION
+
+BIO_s_dgram_pair() returns the method for a BIO datagram pair. A BIO datagram
+pair is similar to a BIO pair (see BIO_s_bio()) but has datagram semantics.
+Broadly, this means that  the length of the buffer passed to a write call will
+match that retrieved by a read call. If the buffer passed to a read call is too
+short, the datagram is truncated or the read fails, depending on how the BIO is
+configured.
+
+The BIO datagram pair attaches certain metadata to each write, such as source
+and destination addresses. This information may be retrieved on read.
+
+A typical application of a BIO datagram pair is to allow an application to keep
+all datagram network I/O requested by libssl under application control.
+
+The BIO datagram pair is designed to support multithreaded use where certain
+restrictions are observed; see THREADING.
+
+The BIO datagram pair allows each half of a pair to signal to other half
+whether they support certain capabilities; see CAPABILITY INDICATION.
+
+BIO_new_bio_dgram_pair() combines the calls to BIO_new(), BIO_make_bio_pair()
+and BIO_set_write_buf_size() to create a connected  pair of BIOs B<bio1>,
```

Comment:
`BIO_new()`, `BIO_make_bio_pair()` and `BIO_set_write_buf_size()` are described on another man page so should be `L<BIO_new(3)>`, etc